source/docs/05-mensural.xml

```diff
@@ -397,6 +397,22 @@
                         <specDesc key="att.plica.vis" atts="dir len"/>
                      </specList>
                   </p>
+                  <figure>
+                     <head>Encoding and rendition of plicas</head>
+                     <graphic url="../images/modules/mensural/ex_plicae.png"/>
+                  </figure>
+               </div>
+               <div xml:id="dots" type="div3">
+                  <head>Dots</head>
+                  <p>Dots of division and augmentation can be encoded by using the <gi scheme="MEI">dot</gi> element. This element is meant to be used as a child of <gi scheme="MEI">layer</gi> following the <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> after which it appears in the original source.</p> 
+                  <p>Dots are not supposed to be encoded as children of notes or rests, but rather as a sibling of these. They are also not supposed to be encoded as attributes (the use of the @dot attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> elements is only acceptable in Common Music Notation, not mensural).</p>
```

Comment:
```suggestion
                  <p>Dots in mensural notation are not encoded as children of notes or rests, but rather as a sibling of these. They are also not encoded as attributes (the use of the @dot attribute in a <gi scheme="MEI">note</gi> or <gi scheme="MEI">rest</gi> elements is only acceptable in Common Music Notation, not mensural).</p>
```

Reply:
Changed to be more direct... "supposed" is like giving a suggestion, but I believe it's actually an error, so they absolutely should not be.